project-fob/Models/Fob.cs

```diff
@@ -1,37 +1,69 @@
-﻿using project_fob.Data;
+﻿using Microsoft.EntityFrameworkCore;
+using project_fob.Data;
 using System;
 using System.Collections.Generic;
 using System.ComponentModel.DataAnnotations;
 using System.Linq;
 
 
-namespace project_fob.Models 
+namespace project_fob.Models
 {
-    public class Fob 
+    public class Fob
     {
         [Key]
         public int Id { get; set; }
-        public int AttendeeCount { get; set; }
-        public int FobCount { get; set; }
+        
+        //This variable and code is mainly here because of past values are bein loaded into the project.
+        public int AttendeeCount
+        {
+            get
+            {
+                if (Meeting != null)
+                { return -1; }
+                else { return -11; }
+            }
+            set { }
+        }
+        public int FobCount { get { return Fobbed.Count; } set { } }
+
+
         public DateTime TopicStartTime { get; set; }
-        public Meeting Meeting { get; set; } 
-        public List<Attendee> fobbed { get; set; } = new List<Attendee>();
+        public Meeting Meeting { get; set; }
+        public List<Attendee> Fobbed { get; set; } = new List<Attendee>();
 
 
         public Fob() { }
-        public Fob(Meeting meeting) 
-        { 
+        public Fob(Meeting meeting)
+        {
             // maybe check if a meeting already has a fob ? also need to check that is active
             Meeting = meeting;
-            fobbed = new List<Attendee>();
-            AttendeeCount = 0;
-            FobCount = 0;
+            Fobbed = new List<Attendee>();
+            //AttendeeCount = 0;
+            //FobCount = 0;
             TopicStartTime = DateTime.Now;
```

Comment:
This is probably never going to matter in real life, but when it's a good habit to always use UTC when storing times for anything - otherwise you can get yourself into a mess. So for, you'd want to use `DateTime.UtcNow` instead of `DateTime.Now` (which always uses the current time zone)